### PR TITLE
Add execution environment list to Organizations

### PR DESCRIPTION
**Pull request by nixocio**
Depends-On: https://github.com/ansible/zuul-jobs/pull/43

Add execution environment list to Organizations

See: https://github.com/ansible/awx/issues/8210

image

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

**Review by jakemcdermott — APPROVED** [review]
Code lgtm and I pulled this down locally and did a bit of testing on the related list. Also lgtm!

File: awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      key={executionEnvironment.id}
+      aria-labelledby={labelId}
+      id={`${executionEnvironment.id} `}
+    >
+      <DataListItemRow>
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell
+              key="image"
+              aria-label={i18n._(t`execution environment image`)}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Organization/OrganizationExecEnvList/OrganizationExecEnvListItem.jsx]: Possibly need some sentence casing here? 
```suggestion
              aria-label={i18n._(t`Execution environment image`)}
```

**Comment by nixocio** [comment]
@jakemcdermott, updated and rebased.

**Review by dsesami — CHANGES_REQUESTED** [review]
A few notes:
1. Sorting by Organization should be removed from this view. It is all the same organization.
2. Clicking the image link does not hyperlink anywhere. It should link to the EE details, and then the backlink should know how to go back to the list of EEs for that org, and not just back to the main list of EEs.

**Comment by nixocio** [comment]
@dsesami, updated per your comments.

**Comment by shanemcd** [comment]
recheck

**Review by dsesami — APPROVED** [review]
The hyperlink works, but the backlink needs to be "smarter". However, this is a problem throughout the application and is out of the scope of this PR.

**Comment by nixocio** [comment]
regate

**Pull request merged**